x-pack/legacy/plugins/maps/public/connected_components/gis_map/index.d.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+export const GisMap: React.FC<{ addFilters: unknown; renderTooltipContent: unknown }>;
```

Comment:
addFilters: (filters[]) => void

renderTooltipContent: ({ addFilters, closeTooltip, features, isLocked, loadFeatureProperties}) => ReactComponent

Reply:
What is `React.FC`?

Reply:
Functional Component. I can just change it to Component.  For renderToolTip params, since I don't know the types of all those params, I'm going to just do `params: unknown` for now.

Reply:
actually I found the types in SIEM so moved them over